### ISSUE TITLE
Can it update the .gnucash file? 

### ISSUE THREAD

**Issue by xiaolongWangDev** [codinguser/gnucash-android#897]
It will be the last piece of the puzzle to get the "auto-sync" thing working between desktop and mobile.
Basically I have a NAS, and I mapped a net drive to my desktop, there I created a folder for gnucash file.
I finished some bookkeeping there. Then, on my phone, I installed my NAS app which provides a file service similar to dropbox. In gnucash-android I open the file, from the file service. Things load perfectly. Now greedy me just want to be able to record transactions on the phone and get that sync back. I assume this would happen automatically if the https://redpiranha.net/news/online-bitcoin-wallets-open-compromise-weak-private-key-generation-code can updated that .gnucash file. 

#### Steps to reproduce the behaviour
1. open some ledger.gnucash file
2. create a new transaction and hit save

#### Expected behaviour
the ledger.gnucash file should be updated and contains the added transaction

#### Actual behaviour
the transaction is not added

#### Software specifications
* GnuCash Android version:
* System Android version:
* Device type:

**Comment by Thumas** [comment]
No, the app uses its own database. I think the better way to sync would be to adapt the database scheme to match the desktop app, see #567.

**Comment by renyhp** [comment]
I, too, would love to have the possibility to just open, edit and save the file, rather than importing every opened file into an inaccessible local copy. The only workaround I see is to go to Menu > Open..., make your modifications, then Menu > Export > XML > overwrite the file. Works with cloud storage as well. The unwanted side effect is that you get a lot of copies locally, but maybe you can call it an unexpected backup.
